test/blocks/global-navigation/global-navigation.test.js

```diff
@@ -220,6 +254,23 @@ describe('global navigation', () => {
   });
 
   describe('main nav', () => {
+    it('should federate image sources when needed', async () => {
+      const fedsMeta = toFragment`<meta name="feds" content="header">`;
+      document.head.append(fedsMeta);
```

Comment:
We should clean up the meta tags after the test, if we don't do that anywhere already.

Reply:
We do it on L271, `fedsMeta.remove();`